### ISSUE TITLE
ClearKey content not playing

### ISSUE THREAD

**Issue by FritzHeiden** [HbbTV-Association/ReferenceApplication#37]
ClearKey content is not playing in either Chrome or Firefox.

```
DRM: unable to create session! --Error generating key request -- TypeError
DOMException: The fetching process for the media resource was aborted by the user agent at the user's request.
```

edit: Tested in current stable: http://refapp.hbbtv.org/production

**Comment by jpiesing** [comment]
@juhajoki

**Comment by juhajoki** [comment]
@FritzHeiden thank for the report. this might be an http/https issue. I've changed CK LA URLs to use https in testing instance, can you re-test from here: https://refapp.hbbtv.org/testing/catalogue/

**Comment by jpiesing** [comment]
Clearkey works for me - Firefox, Chrome, Edge.

**Comment by jpiesing** [comment]
It looks like there are significant mixed content issues.

If the refapp is loaded from an http:// URL then EME may fail due to mixed content restrictions (will fail on modern browsers). Realistically, I think the reference app should always load from an https:// URL.

If the refapp is loaded from an https:// URL then that fails because dash.js is loaded from an http:// URL.

![image](https://user-images.githubusercontent.com/8490792/108259925-f8827e00-7161-11eb-968d-f9c885922aec.png)

To summarise;

I suggest we consider either
1) changing it to always load from an https:// URL (http:// re-directs to https://) and to load dash.js from an https:// URL or 
2) the code to load dash.js should check if the page was loaded from http:// or https:// and load dash.js appropriately and the UI be modified to disable DRM when in MSE-EME mode.

**Comment by juhajoki** [comment]
option 1 would be straight-forward to implement, but we need to keep in mind that currently refapp.hbbtv.org is using Let's Encrypt certificate, which some (although very old) devices might not support.

option 2 seems to me to be unnecessary work just to keep http as an option. 

I "vote" to move https only for all assets using the current certificate in place. Impact would be minimal.

**Comment by bobcampbell-eurofins** [comment]
Conclusion of IITF is to move forward with IITF and document the fact a very small number of devices can't handle the Lets Encrypt certificate (which is a device problem that rises with this instance of the RefApp and isn't a refapp problem as such)

Plus add some documentation on what keys to use when testing on PC would be helpful!

**Comment by juhajoki** [comment]
https now default for all assets in staging and testing instances
